src/saveload/saveload.cpp

```diff
@@ -2920,7 +2920,12 @@ static inline void ClearSaveLoadState()
  */
 static void SaveFileStart()
 {
-	_sl.game_speed = _game_speed;
+	/**
+	 * If fast forward is via key, save normal game speed.
+	 * If the key continues to be pressed, the graphics driver will continue to fast forward the game.
+	 * If the key is released during autosave, the normal speed will be restored.
+	 */
+	_sl.game_speed = VideoDriver::GetInstance()->isFastForwardViaKey() ? 100 : _game_speed;
```

Comment:
```suggestion
	_sl.game_speed = VideoDriver::GetInstance()->IsFastForwardViaKey() ? 100 : _game_speed;
```
Follow-up of the other suggested change.